js/site-orientation.js

```diff
@@ -9,6 +9,14 @@ var helpers = require('./helpers');
 var uri = window.location.toString();
 var uriQuery = helpers.sanitizeQueryParams(URI.parseQuery(window.location.search));
 
+// Use a fresh localstorage item once we're in production
+var STORAGE_ITEM;
+if (window.CANONICAL_BASE === 'https://fec.gov') {
```

Comment:
This is just about what we want to set the canonical URL as. Are we setting it to www.fec.gov and redirecting from fec.gov to that? Or vice-versa?